### PR TITLE
Fixing build compatibility with docker-compose v2

### PR DESCRIPTION
**Pull request by davidferlay**
Composer v2 doesn't support spaces in .env variables as described here https://github.com/docker/compose/issues/8763#issuecomment-961913249

```
➜  skilld-docker-container git:(master) make all
key cannot contain a space
```

### CODE REVIEW

File: .env.default [hunk 1]
```diff
@@ -16,8 +14,6 @@ IMAGE_SOLR=solr:8-slim
 IMAGE_REDIS=redis:5-alpine
 IMAGE_DRIVER=zenika/alpine-chrome
 CLEAR_FRONT_PACKAGES=no
-ADD_PHP_EXT=graphicsmagick php7-intl
-#ADD_PHP_EXT=php7-pdo_pgsql postgresql-client postgresql-contrib gnu-libiconv wkhtmltopdf php7-redis
```
Comment (andypost) [inline .env.default:20]: Please keep this line as it helpful to maintain a list of most-used packages

Comment (davidferlay) [reply .env.default:20]: This remains here https://github.com/skilld-labs/skilld-docker-container/pull/401/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L44

**Review by andypost — COMMENTED** [review]
sadly I missed to publish review, at least `php7-intl` is not needed

File: Makefile [hunk 2]
```diff
@@ -42,6 +42,10 @@ php = docker-compose exec -T --user $(CUID):$(CGID) php ${1}
 # Execute php container as root user
 php-0 = docker-compose exec -T --user 0:0 php ${1}
 
+ADDITIONAL_PHP_PACKAGES := tzdata graphicsmagick php7-intl #php7-redis wkhtmltopdf gnu-libiconv php7-pdo_pgsql postgresql-client postgresql-contrib
```
[the remark this answers is not in the document]
Comment (andypost) [reply Makefile:45]: Fixed via 3f16337cb67e3f08504cc79339e7066c67daf1ed

**Pull request merged**